compression/distortion.h

```diff
@@ -25,6 +25,8 @@ namespace gcpp {
 class DistortionStats {
  public:
   void Notify(float original, float distorted) {
+    (void)padding_;  // prevent unused member warning
```

Comment:
Maybe it would be better to use `HWY_MAYBE_UNUSED`

Reply:
Unfortunately GCC seems to turn that into another warning:
```
class Test {
    HWY_MAYBE_UNUSED int member;
};
```
```
<source>:11:26: warning: 'unused' attribute ignored [-Wattributes]
   11 |     HWY_MAYBE_UNUSED int member;
      |                          ^~~~~~
```